Source/Core/DolphinQt2/Config/Graphics/GraphicsSlider.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 Dolphin Emulator Project
```

Comment:
@spycrab Well, what do you think of the merits I described above? Do you not think those are good criteria for making design choices?